public/app/percona/integrated-alerting/components/Alerts/Alerts.tsx

```diff
@@ -124,24 +124,56 @@ export const Alerts: FC = () => {
 
   const renderSelectedSubRow = React.useCallback((row: Row<Alert>) => <pre>{row.original.rule?.expr}</pre>, []);
 
+  const handleSilenceAll = useCallback(async () => {
+    await AlertsService.toggle({ silenced: 'TRUE', alert_ids: [] });
```

Comment:
Can this be a single function that receives TRUE or FALSE as parameter? It can still have the name `handleSilenceAll`

Reply:
good point, thanks!